pkg/daemon/update.go

```diff
@@ -66,6 +66,14 @@ var (
 	nonFCOSAuthKeyPath     = filepath.Join(coreUserSSHPath, "authorized_keys")
 )
 
+type ImageUpdater struct {
```

Comment:
Move these structs and their methods into a separate file (e.g., `updaters.go`) for clarity.

Reply:
I didn't do this or rename the receiver to minimize the diff. That is definitely not how I would choose to do it, but this PR is against layering and we're still trying to minimize the impact of layering vs master. That said I would much prefer to do it right if you think we can get it in